jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/settlerimages/SettlerImageMap.java

```diff
@@ -346,6 +346,19 @@ public Image getImageForSettler(EMovableType movableType, EMovableAction action,
 	 */
 	private SettlerImageMapItem getMapItem(EMovableType movableType,
 			EMovableAction action, EMaterialType material, EDirection direction) {
+
+
+		if(movableType == null
+				|| action == null
+				|| material == null
+				|| direction==null) {
+			int i = 0;
+			i++;
```

Comment:
What is the idea of this? You declare a local variable i and increment it by one and then nothing happens with it?